bundles/org.openhab.binding.logreader/.classpath

```diff
@@ -6,12 +6,7 @@
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="src" output="target/classes" path="src/3rdparty/java">
-		<attributes>
-			<attribute name="optional" value="true"/>
-			<attribute name="maven.pomderived" value="true"/>
-		</attributes>
-	</classpathentry>
+	<classpathentry kind="src" path="src/3rdparty/java"/>
```

Comment:
no output path?

Reply:
It'll use the default which is `target/classes`